### PR TITLE
New Burning Sprite

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR adds a new burning sprite.

## changes
- adds a new burning sprite.

## screenshots
<details>
<summary>🖼️ click to view</summary>

image
**figure 1:** the burning sprite.
</details>

## videos
<details>
<summary>🎥 click to view</summary>

https://user-images.githubusercontent.com/99297919/202787576-f9c72d91-b89a-44ea-96ea-be4e3bc4488d.mp4

**figure 1:** the burning sprite.
</details>

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by Butterrobber202** [comment]
this is going to make my war crimes so much more immersive.

**Review by Alberyk — APPROVED** [review]

**Comment by Zelmana** [comment]
Certified Thich Quang Duc moment

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -55,8 +55,11 @@
 	var/show_ssd = "fast asleep"
 	var/short_sighted
 	var/bald = 0
-	var/light_range
-	var/light_power
+
+	// Light
+	var/light_range = null
+	var/light_power = null
+	var/light_color = null
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/species/species.dm:62]: these are completely unused, so remove them

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/species.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/modules/mob/living/carbon/human/species/species.dm]: done.

Comment (SleepyGemmy) [reply code/modules/mob/living/carbon/human/species/species.dm:62]: they are used by IPCs.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**